Boards.h

```diff
@@ -560,6 +560,26 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define PIN_TO_PWM(p)           (p)
 #define PIN_TO_SERVO(p)         (p)
 
+// Pinoccio Scout
+#elif defined(ARDUINO_PINOCCIO)
+#define TOTAL_ANALOG_PINS       8
+#define TOTAL_PINS              NUM_DIGITAL_PINS // 32
+#define VERSION_BLINK_PIN       23
+#define PIN_SERIAL1_RX          0
+#define PIN_SERIAL1_TX          1
+#define PIN_SERIAL2_RX          13
+#define PIN_SERIAL2_TX          14
+#define IS_PIN_DIGITAL(p)       (((p) >= 2) && ((p) <= 8)) || (((p) >= 24) && ((p) <= 31))
```

Comment:
Do pins 0 and 1 map to Serial or Serial1? If they map to Serial1, the you should be able to use pins 0 and 1 as digital inputs unless those pins don't have digital I/O for some other reason.